assisted-events-scrape/process/host.py

```diff
@@ -0,0 +1,66 @@
+
+class Host:
+    to_summarize = {
```

Comment:
nit: the variable should be in upper case for f it's used as a const, and prefixed with `_` as it looks private too